docs/advanced/flash.md

```diff
@@ -56,6 +56,10 @@ Bootloader-ready declaration of flash VTOR address:
 #endif
 ```
 
+## Start application
+
+The ``mbed_start_application`` is currently implemented only for Cortex-M3/M4/M7. The implementation is located in [the mbed_application code file](https://github.com/ARMmbed/mbed-os/blob/master/platform/mbed_application.c). If your target is not supported, you must implement this function in the target HAL.
```

Comment:
I'd like to change this paragraph to active voice. Would it be correct to say "The `mbed_start_application` implementation exists only for Cortex-M3, Cortex-M4 and Cortex-M7. You can find it in [the mbed_application code file](https://github.com/ARMmbed/mbed-os/blob/master/platform/mbed_application.c). If `mbed_start_application` does not support your target, you must implement this function in the target HAL."?